recipes/seadex-essentials/all/test_package/conanfile.py

```diff
@@ -0,0 +1,33 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps
+from conan.tools.build import can_run
+from conan.tools.cmake import cmake_layout
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    test_type = "explicit"
+
+    def layout(self):
+        cmake_layout(self)
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+        self.requires("spdlog/1.11.0")
+        self.requires("fmt/9.1.0")
```

Comment:
Since `test_package.cpp` doesn't include headers from these directly, you may need to mark them as transitive in the requires statements of the `conanfile.py` by adding the arguments `transitive_headers=True` and/or `transitive_libs=True`. They should not show up here in the test package.

```suggestion
```